Components/AppTabNavigator/StocksTab.js

```diff
@@ -1,135 +1,123 @@
-import React, { Component } from "react";
-import { Card } from 'react-native-paper';
+import React, {Component} from 'react';
+import {Card} from 'react-native-paper';
 
-import {
-    View,
-    Text,
-    StyleSheet
-} from "react-native";
+import {View, StyleSheet} from 'react-native';
 
-import { Container, Content, Icon } from 'native-base'
-import { useNavigation } from "@react-navigation/native";
+import {Container, Content, Icon} from 'native-base';
 
 class StocksTab extends Component {
-    static navigationOptions = {
-        tabBarIcon: ({ tintColor }) => (
-            <Icon name="android-home" style={{ color: tintColor }} />
-        ) 
+  static navigationOptions = {
+    tabBarIcon: ({tintColor}) => (
+      <Icon name="android-home" style={{color: tintColor}} />
+    ),
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      isLoading: false,
+      // companyInfo is one big object holding the arrays of company information...might need to change this behavior to maybe having an array of "Company" objects each
+      // with their own individual properties.
+      companyInfo: {
+        companyNames: [],
+        companySymbols: [],
+      },
+    };
+  }
+
+  componentDidMount() {
+    //NetInfo.isConnected.fetch().then(isConnected => {
+    if (true) {
+      this.getStockCardData();
     }
+  }
 
-    constructor(props) {
-        super(props);
-        this.state = {
-            isLoading: false,
-            // companyInfo is one big object holding the arrays of company information...might need to change this behavior to maybe having an array of "Company" objects each 
-            // with their own individual properties.
-            companyInfo: {
-                companyNames: [],
-                companySymbols: []
-            }
-        };
-    }
+  async getStockCardData() {
+    // Hard coded api_key. Not sure if we will need to change this.
+    let api_key = 'Tpk_77a598a1fa804de592413ba39f6b137a';
```

Comment:
Harding API keys should always be avoided. We can resolve this in a different PR. I'll have to research the proper way to handle API keys in react native.